src/FileSystem/FilesFinder.php

```diff
@@ -86,6 +86,11 @@ public function findInDirectoriesAndFiles(array $source, array $suffixes): array
             $smartFileInfos[] = new SmartFileInfo($file);
         }
 
+        $composerPath = getcwd() . '/composer.json';
```

Comment:
Would move this to the FileFactory https://github.com/rectorphp/rector/blob/main/src/ValueObjectFactory/Application/FileFactory.php#L39

Reply:
Maybe, it is one level up. What would be the difference?